contracts/Commitment/Commitment.sol

```diff
@@ -342,6 +342,14 @@ contract Commitment is
         return MIN_TICKET_EUR;
     }
 
+    function ethEurFraction()
```

Comment:
Why just not call `convertToEur` with  1 ether? If you persist on having this (there may be numeric precision problems due to mul/div) please add test for getter here:
```
it("should deploy", async () => {
      await prettyPrintGasCost("Neumark deploy", neumark);
--> here
    });
```
in `Neumark.js`
later I'll merge audit results with other getters' tests